### PR TITLE
fix: fp when editing template-parts in gutenberg editor

### PR DESCRIPTION
**Pull request by EsadCetiner**
Fixes false positives with the new gutenberg when editing template parts of a website, for instance the header or footer.

~~I couldn't figure out why the linter is failing, I can't find any trailing whitespace, missing/extra line breaks, and the error message is ambiguous.~~ nvm I can see the error now after I've opened the PR.

### CODE REVIEW

**Comment by azurit** [comment]
Can you 'fix' tests?
 - we use `"OWASP CRS test agent"` as User-Agent for CRS (so we should do it also for plugins)
 - URI should start with endpoint `/method` (i.e. `/post/...`)
 - explicitly set protocol version using `version: "HTTP/1.1"`

Thanks.

**Comment by EsadCetiner** [comment]
@azurit 

Thanks, I missed that, cleaned up the tests a bit too.

File: tests/regression/wordpress-rule-exclusions-plugin/9507600.yaml [hunk 1]
```diff
@@ -13,10 +13,11 @@ tests:
             dest_addr: 127.0.0.1
             headers:
               Host: localhost
-              User-Agent: OWASP CRS
+              User-Agent: OWASP CRS test agent
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             port: 80
             method: GET
+            version: "HTTP/1.1"
             uri: /wp-admin/user-edit.php?user_id=9&wp_http_referer=%2Fwp-admin%2Fusers.php%3Fupdate%3Dadd%26id%3D9
```
Comment (azurit) [inline tests/regression/wordpress-rule-exclusions-plugin/9507600.yaml:21]:
```suggestion
            uri: /get/wp-admin/user-edit.php?user_id=9&wp_http_referer=%2Fwp-admin%2Fusers.php%3Fupdate%3Dadd%26id%3D9
```

File: tests/regression/wordpress-rule-exclusions-plugin/9507971.yaml [hunk 2]
```diff
@@ -45,11 +45,11 @@ tests:
             dest_addr: 127.0.0.1
             headers:
               Host: localhost
-              User-Agent: OWASP CRS
+              User-Agent: OWASP CRS test agent
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             port: 80
             method: GET
-            uri: /wp-admin/plugin-install.php
+            uri: /get/wp-admin/plugin-install.php
```
Comment (azurit) [inline tests/regression/wordpress-rule-exclusions-plugin/9507971.yaml:52]:
```suggestion
            version: "HTTP/1.1"
            uri: /get/wp-admin/plugin-install.php
```

File: tests/regression/wordpress-rule-exclusions-plugin/9507971.yaml [hunk 3]
```diff
@@ -29,11 +29,11 @@ tests:
             dest_addr: 127.0.0.1
             headers:
               Host: localhost
-              User-Agent: OWASP CRS
+              User-Agent: OWASP CRS test agent
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             port: 80
             method: GET
-            uri: /wp-admin/plugins.php
+            uri: /get/wp-admin/plugins.php
```
Comment (azurit) [inline tests/regression/wordpress-rule-exclusions-plugin/9507971.yaml:36]:
```suggestion
            version: "HTTP/1.1"
            uri: /get/wp-admin/plugins.php
```

File: tests/regression/wordpress-rule-exclusions-plugin/9507971.yaml [hunk 4]
```diff
@@ -13,11 +13,11 @@ tests:
             dest_addr: 127.0.0.1
             headers:
               Host: localhost
-              User-Agent: OWASP CRS
+              User-Agent: OWASP CRS test agent
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             port: 80
             method: GET
-            uri: /wp-admin/plugin-install.php
+            uri: /get/wp-admin/plugin-install.php
```
Comment (azurit) [inline tests/regression/wordpress-rule-exclusions-plugin/9507971.yaml:20]:
```suggestion
            version: "HTTP/1.1"
            uri: /get/wp-admin/plugin-install.php
```

**Comment by EsadCetiner** [comment]
@azurit It caused a false positive, 9507600 is meant to apply unconditionally but it only apples for wp-admin because of the skip rule.

**Comment by azurit** [comment]
Then i suggest to move 9507600 out of that block (and renumber it if you feel it's needed).

**Review by azurit — APPROVED** [review]

**Pull request merged**